testing/nginx-naxsi/nginx.initd

```diff
@@ -1,42 +1,69 @@
 #!/sbin/openrc-run
 
-extra_started_commands="reload"
-extra_commands="configtest"
+description="Nginx http and reverse proxy server"
+extra_started_commands="reload reopen upgrade"
+
+cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
+pidfile=/run/nginx/nginx.pid
+tmpdir=/var/lib/nginx/tmp
+command=/usr/sbin/nginx
+command_args="-c $cfgfile"
+required_files="$cfgfile"
 
 depend() {
-	need net
+#	need net => breaks LXC containers ##
```

Comment:
Aha, I don’t use statically assigned address, but DHCP. However, there are dozens of runscripts with `need net`, so if this is an issue on LXC, then it must be solved in OpenRC, not just in single runscript. Please open an issue on http://bugs.alpinelinux.org/.